datafusion/physical-expr/src/window/window_expr.rs

```diff
@@ -559,7 +559,6 @@ pub enum NthValueKind {
 
 #[derive(Debug, Clone)]
 pub struct NthValueState {
-    pub range: Range<usize>,
```

Comment:
field `range` is never read. Not sure if it will still be used in the future. cc @mustafasrepo